fastlane/actions/plugin_scores.rb

```diff
@@ -0,0 +1,54 @@
+module Fastlane
+  module Actions
+    class PluginScoresAction < Action
+      def self.run(params)
+        require_relative '../helper/plugin_scores_helper.rb'
+        require "erb"
+
+        plugins = fetch_plugins.sort_by { |v| v.data[:overall_score] }.reverse
+
+        result = plugins.collect do |current_plugin|
+          @plugin = current_plugin
+          result = ERB.new(File.read(params[:template_path]), 0, '-').result(binding) # http://www.rrn.dk/rubys-erb-templating-system
+        end.join("\n")
+
+        File.write(File.join("docs", params[:output_path]), result)
+      end
+
+      def self.fetch_plugins
+        page = 1
+        plugins = []
+        loop do
+          url = "https://rubygems.org/api/v1/search.json?query=fastlane-plugin-&page=#{page}"
```

Comment:
would it make sense to throttle this? I feel like hitting this api as fast as we can might not be good. I guess it depends how many pages there are.